### PR TITLE
Update prevent-changes-to-security-settings-with-tamper-protection.md

### PR DESCRIPTION
**Pull request by danorum**
Added a note in the applies to section regarding the requirement to use the new unified solution package that is currently in preview. I am not sure if it belongs here in Applies to: or in the Requirements section further down the page.

### CODE REVIEW

**Comment by danorum** [comment]
Actually, that should not have Windows Server 2016 as it does have tamper protection without the new unified package. I am unsure how to change my edit/pull request to reflect that.

**Comment by danorum** [comment]
I believe I fixed it to take Windows Server 2016 out with an additional Commit.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Tamper protection is available for devices that are running one of the following
 - Windows Server 2016
 - Windows Server 2012 R2
 
+[!NOTE]Tamper protection on Windows Server 2012 R2 is available for devices onboarded using the modern unified solution package. For more information, see https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/configure-server-endpoints.md#new-functionality-in-the-modern-unified-solution-for-windows-server-2012-r2-and-2016-preview
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md]:
```suggestion
> [!NOTE]
> Tamper protection in Windows Server 2012 R2 is available for devices onboarded using the modern unified solution package. For more information, see [New functionality in the modern unified solution for Windows Server 2012 R2 and 2016 Preview](/microsoft-365/security/defender-endpoint/configure-server-endpoints?view=o365-worldwide#new-functionality-in-the-modern-unified-solution-for-windows-server-2012-r2-and-2016-preview).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @danorum, kindly commit the suggested change. Many thanks.

**Pull request merged**